spec/fixture_app/config/application.rb

```diff
@@ -1,5 +1,6 @@
 module Fixture
   class Application < Rory::Application
     turn_off_request_logging!
+    filter_parameters :orcas, :noodles
```

Comment:
That's a good question.  I do like this convention, yes, but I'm not stuck on it, and you make a good point about what is expected.  I myself like the idea that (with named parameters) we're moving towards an expectation that multiple unnamed parameters are a collection.